File: Assignment No.22/Program1.c
/* Problem Statement : Accept Number of rows and column from user and display below Pattern.
Input : iRow - 4            iCol - 4
Output :
			*
			*  *
			*  *  *
			*  *  *  *
*/
#include<stdio.h>

void Pattern(int iRow,int iCol)
{
	int i=0,j=0;
	for(i=0;i<iRow;i++)
	{
		for(j=0;j<iCol;j++)
		{
			if((i==j)||(i>j))
			{
				printf("*\t");
			}
		}
		printf("\n");
	}
}

int main()
{
	int iValue1 = 0,iValue2 = 0;

	printf("Enter number of rows and columns :");
	scanf("%d%d",&iValue1,&iValue2);

	Pattern(iValue1,iValue2);

	return 0;
}

File: Assignment No 10/Program1.java
//Problem Statement : Write a Program which accept range from user and display all number in that range
//Input : 23 35
//Output : 23 24 25 26 27 28 29 30 31 32 33 34 35
// Input : -10 2
//Output : -10 -9 -8 -7 -6 -5 -4 -3 -2 -1 0 1 2 
// Input : 90 18
//Output :  Invalid range 
import java.io.*;

class Program1
{
	public static void RangeDisplay(int iStart,int iEnd)
	{
		if(iStart>iEnd)
		{
			System.out.println("Invalid Range");
		}
		for(;iStart<=iEnd;iStart++)
		{
			System.out.print(iStart);
			System.out.print("\t");
		}
	}
	public static void main(String args[]) throws Exception
	{
		int iValue1=0,iValue2=0;
		BufferedReader bobj= new BufferedReader(new InputStreamReader(System.in));

		System.out.println("Enter Starting Point");
		iValue1=Integer.parseInt(bobj.readLine());


		System.out.println("Enter Ending Point");
		iValue2=Integer.parseInt(bobj.readLine());

		RangeDisplay(iValue1,iValue2);
	}
}